assets/js/blocks/cart-checkout/feedback-prompt/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Component to render a Feedback prompt in the sidebar.
+ */
+const FeedbackPrompt = ( { text } ) => {
+	return (
+		<div className="wc-block-feedback-prompt">
+			<Gridicon icon="comment" />
+			<h2 className="wc-block-feedback-prompt__title">
+				{ __( 'Feedback?', 'woo-gutenberg-products-block' ) }
+			</h2>
+			<p className="wc-block-feedback-prompt__text">{ text }</p>
+			<a
+				href="https://wordpress.org/support/plugin/woo-gutenberg-products-block/reviews/"
```

Comment:
Have we verified this is where we want feedback to go?

Reply:
AFAIK it's not decided yet, so I added this link as a placeholder. I was planning to ask @garymurray when he comes back from AFK. For now, I added a `@todo` comment so we don't forget about it (b88391af4edfc8ff36bb72ac538162ad174de145).